phpBB/styles/prosilver/theme/links.css

```diff
@@ -73,6 +73,18 @@ a.topictitle {
 	text-decoration: none;
 }
 
+.topic-title {
+	font-family: "Trebuchet MS", Helvetica, Arial, sans-serif;
+	font-size: 16px;
+	font-weight: bold;
+	text-decoration: none;
+}
+
+.title-read,
+.title-unread {
+	display: inline;
+}
+
```

Comment:
These don't appear to be specifically about link formatting and should be moved to cp.css